### DISCUSSION TITLE
Unable to detect device share. This may be due to browser security settings. Adjust your current browser settings from 'Strict' to 'Moderate'. Alternatively, login via a Chrome Browser.

### DISCUSSION THREAD

**Discussion by ExtracySolutions** [Web3Auth/Web3Auth#1130] [Help]
**Unable to detect device share. This may be due to browser security settings. Adjust your current browser settings from 'Strict' to 'Moderate'. Alternatively, login via a Chrome Browser.**

I implement with tkey self host + react web, What i'm i missing?

![image](https://user-images.githubusercontent.com/113089637/209526198-37fd5656-8fc5-46fa-bbc8-cec17020aa99.png)
<img width="646" alt="image" src="https://user-images.githubusercontent.com/113089637/209526350-01d14920-a754-4ed5-9b12-55cb06d44c5f.png">

<img width="708" alt="image" src="https://user-images.githubusercontent.com/113089637/209526374-176cf09b-6f2a-4aee-8135-e7bc51e953b2.png">

===> here is my log
![image](https://user-images.githubusercontent.com/113089637/209538425-b6991147-5c60-4381-a35e-129b676294c4.png)

**Comment by shahbaz17** [comment]
Hey @ExtracySolutions 

Please take a look at https://github.com/Web3Auth/examples/tree/main/self-host/self-host-react-example as a reference.

It looks like a config mismatch. Also, look at the latest documentation on Self Host https://web3auth.io/docs/sdk/self-host/

**Reply by ExtracySolutions** [reply]
@shahbaz17  i have gone through all of your docs and don't see a specific instruction for self host + auth0, the code above i believe is missing a few fields, so can you point out exactly what i am missing or a specific tutorial for self host + auth0
